Documentation/op-guide/monitoring.md

```diff
@@ -56,6 +56,15 @@ nohup /tmp/prometheus \
 Now Prometheus will scrape etcd metrics every 10 seconds.
 
 
+## Alerting
+
+There are two sets of alerts as the exposed metrics have been largely reworked in etcd v3.
+
+* [etcd v2 alerts](./etcd2.rules)
```

Comment:
remove etcd v2. if you want to have this, you can put this into https://github.com/coreos/etcd/tree/master/Documentation/v2